ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateProcessor.java

```diff
@@ -137,17 +137,7 @@ public void onNewSlot(Date date) throws StateTransitionException, InterruptedExc
             + "  |  "
             + nodeSlot.longValue() % Constants.GENESIS_SLOT);
 
-    synchronized (this.store.getSyncObject()) {
```

Comment:
What this was or how it worked always confused me lol, we should change the 3000 milliseconds to Constants.SECONDS_PER_SLOT / 2 tho